test/k8s-integration/driver-config.go

```diff
@@ -124,3 +138,38 @@ func generateDriverConfigFile(platform, pkgDir, storageClassFile, snapshotClassF
 
 	return configFilePath, nil
 }
+
+func getGKEClusterVersion() (major, minor int, err error) {
```

Comment:
I remember @saikat-royc had already implemented some utilities that did this?

Reply:
Yup, thanks, I used them instead.